src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java/JavaLanguageLexer.g4

```diff
@@ -116,12 +116,12 @@ tokens {
     STRING_TEMPLATE_CONTENT, EMBEDDED_EXPRESSION_BEGIN, EMBEDDED_EXPRESSION,
     EMBEDDED_EXPRESSION_END,
 
```

Comment:
had to leave string templates token in the lexer due to: 
```
[ERROR]   GeneratedJavaTokenTypesTest.testTokenNumbering:741 A token's number has changed. Please open 'GeneratedJavaTokenTypesTest' and confirm which token is at fault.
Token numbers must not change or else they will create a conflict with users.
See Issue: https://github.com/checkstyle/checkstyle/issues/505
```
If we remove string template tokens from the lexer the last two tokens' numbers will change and we have a test that ensures that the tokens' numbers don't change. 

accordingly, coverage for javaLnaguageLexer.java will drop 

@nrmancuso please help in this how to remove the tokens and pass GeneratedJavaTokenTypesTest.testTokenNumbering ?